src/apps/ipv6/nd_light.lua

```diff
@@ -77,8 +77,12 @@ local function check_ip_address(ip, desc)
 end
 
 function nd_light:new (arg)
-   local conf = arg and config.parse_app_arg(arg) or {}
-   local o = nd_light:superClass().new(self)
+   local arg = arg and config.parse_app_arg(arg) or {}
+   local conf = {}
+	for k,v in pairs(arg) do
+		conf[k] = v
+	end
+	local o = nd_light:superClass().new(self)
```

Comment:
I removed the tabs and added a comment on why that change is needed: basically if you modify the arg table directly, you won't be able to reuse it and when the app gets reconfigured it crashes or something like that IIRC.